api/src/main/java/org/openmrs/parameter/OrderSearchCriteria.java

```diff
@@ -50,8 +50,8 @@
 	 * Instead of calling this constructor directly, it is recommended to use {@link OrderSearchCriteriaBuilder}.
 	 * @param patient the patient the order is for
 	 * @param careSetting the care setting to match on
-	 * @param concepts the concepts defining the order must be in this collection
-	 * @param orderTypes the order types to match on must be in this collection
+	 * @param concepts the concepts defining the order must be in this collection if specified, matches on all concepts otherwise
+	 * @param orderTypes the order types to match on must be in this collection if specified, matches on all order types otherwise
```

Comment:
Same here **otherwise** seems to have been left hanging here

Reply:
@wluyima similarly here, "otherwise" can end a sentence.